### PR TITLE
Shows default image in the frontend

### LINKED ISSUE #82: Create and use placeholder image for recipes without a photo.
Possibly based of the image used in the `favicon`. @RyanNoelk is there an original somewhere?

### PR DESCRIPTION
**Pull request by hermzz**
Because finding a new logo image is outside of the scope of this PR I've just created a placeholder for the time being.

Fixes #82.

### CODE REVIEW

File: frontend/modules/recipe/components/Recipe.js [hunk 1]
```diff
@@ -213,6 +208,22 @@ var RecipeScheme = injectIntl(React.createClass({
...
+      return (
+        <img className="banner-img img-responsive hidden-xs" src={ this.props.data.photo } />
+      );
+    }
+  },
+
+  showRecipeImageThumb: function() {
+    if (this.props.data.photo_thumbnail) {
+      return (
+        <img className="img-responsive" src={ this.props.data.photo_thumbnail } />
```
Comment (RyanNoelk) [inline frontend/modules/recipe/components/Recipe.js:230]: I think we should also show the default image here as well. Just doesn't look as good without it, IMO.

**Review by RyanNoelk — CHANGES_REQUESTED** [review]
We should also drop the default image: `api/base/default.png` and the management command folder: `api/base/management`. We are not using them anymore

File: frontend/modules/recipe/components/Recipe.js [hunk 2]
```diff
@@ -126,17 +126,12 @@ var RecipeScheme = injectIntl(React.createClass({
...
-      photo_placeholder: {
-        id: 'recipe.photo_placeholder',
-        description: 'Photo placeholder',
-        defaultMessage: 'Generic placeholder thumbnail'
-      }
     });
 
     return (
       <div className="recipe-details">
         <h1 className="title hidden-xs">{this.props.data.title}</h1>
```
Comment (RyanNoelk) [inline frontend/modules/recipe/components/Recipe.js]: Should we only show this header if there is an Image? Looks weird to have it twice so quickly. Thoughts?

Comment (hermzz) [reply frontend/modules/recipe/components/Recipe.js]: Good point, but now that I've had a good look, I'm not convinced the current layout makes a lot of sense. If the image exists, we show the image twice, which looks kinda weird. And if there is no image, then we show the title of the recipe in a different place and a useless placeholder. I feel like we should either just show the large image first and no first header or thumbnail. Or just the thumbnail (with a placeholder if no image) and have the green title always be the header. Or maybe the large image appears under the green title? 

I'm going to remove the first title if the image doesn't exist for the time being, and we can open a separate issue to see if we can improve this page.

**Review by RyanNoelk — COMMENTED** [review]
I'm still seeing the default image in the api `api/base/default.png`. I'm not 100% sure of the license for that, we should drop it.

Otherwise LGTM!

File: frontend/modules/recipe/components/Recipe.js (hunk 2 above)
Comment (RyanNoelk) [reply frontend/modules/recipe/components/Recipe.js]: I'm by no means a UX/UI expert. If you have some ideas, I'd love to see them

**Comment by hermzz** [comment]
Good point, deleted!

**Comment by RyanNoelk** [comment]
Maybe I'm crazy but I'm not seeing the image as deleted. Did you forget to push a commit?

**Comment by hermzz** [comment]
Not crazy, I'm a moron. Commit pushed for reals this time.

**Review by RyanNoelk — APPROVED** [review]

**Pull request merged**